src/components/dialog/index.js

```diff
@@ -157,7 +157,7 @@ class Dialog extends BaseComponent {
       <TouchableWithoutFeedback>
         <SafeAreaView style={{flexGrow: 1}}>
           {this.props.children}
-          {useSafeArea && Constants.isIphoneX && bottom && <View style={{height: bottomInsets}}/>}
+          {useSafeArea && Constants.isIphoneX && bottom && <View style={{marginTop: bottomInsets}}/>}
```

Comment:
out of curiosity, why did u change to `marginTop`?

Reply:
I do not know why, but the `Dialog` did not respond anymore after opening it on iPhoneX (the application seemed stuck)